packages/ui-components/src/components/select-option/select-option.tsx

```diff
@@ -1,12 +1,15 @@
-import { Component, Host, h, Prop, EventEmitter, Event } from '@stencil/core';
+import { Component, Element, Host, h, Prop, EventEmitter, Event } from '@stencil/core';
 import { ISelectOption, ISelectOptionEvents } from './select-option.types';
+import { isNil } from 'lodash';
 
 @Component({
 	tag: 'kv-select-option',
 	styleUrl: 'select-option.scss',
 	shadow: true
 })
 export class KvSelectOption implements ISelectOption, ISelectOptionEvents {
+	@Element() el!: HTMLKvSelectOptionElement;
+
 	/** @inheritdoc */
 	@Prop({ reflect: true }) label!: string;
```

Comment:
Should we have a prop with the "bottom" and "right" elements?
Otherwise, how would we add a "bottom" or "right" element when using, for example, the `kv-single-select-dropdown` component?